cmd/temporalite/main.go

```diff
@@ -125,6 +128,12 @@ func buildCLI() *cli.App {
 					EnvVars: nil,
 					Value:   nil,
 				},
+				&cli.StringFlag{
+					Name:    configFileFlag,
+					Usage:   `path to tls key`,
+					EnvVars: nil,
```

Comment:
To match upstream flag:

```suggestion
					Aliases: []string{"c"},
					Usage:   "config dir path",
					EnvVars: []string{temporalconfig.EnvKeyConfigDir},
```

As you've already done we can omit a default value, since a config file isn't needed for Temporalite.